vscode/snippets/zeek.json

```diff
@@ -0,0 +1,83 @@
+{
+  "event": {
+    "prefix": "ev",
```

Comment:
Just as a note, the server will complete event identifiers after typing `event X` where `X` preselects matching event names. Currently that doesn't work well together with this snippet, but the snippet is still an improvement if one knows what roughly what one wants.